docs/index.rst

```diff
@@ -3,6 +3,7 @@
   :hidden:
 
   gcloud-api
+  datastore-api
```

Comment:
Maybe `datastore-overview`?  If it is just the information you added back, that seems descriptive enough.